.changes/unreleased/tinymce-TINY-13029-2025-11-28.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
```

Comment:
This is not an editor feature. If there isn't an oxide-components changelog - and I don't expect there to be one at this stage - changes to it aren't recorded.

- [ ] remove